UPDATING-K8S.md

```diff
@@ -0,0 +1,35 @@
+# Updating Kubernetes
+
+## go.mod
+
+- Update `k8s.io/` replace directives
+  - Most are `v0.X.y`, but `k8s.io/kubernetes` is `v1.X.y`
+  - Also update `k8s.io/kube-openapi` based on `go mod graph | grep ' k8s.io/kube-openapi' | grep '^k8s.io/'`
```

Comment:
👍 nice that there's an automatedish wa to do this - I mostly just guessed and tested 😓